mintpy/prep_fringe.py

```diff
@@ -127,7 +135,9 @@ def read_vrt_info(vrt_file):
     # source dir
     type_tag = root.find(prefix + '/SourceFilename')
     src_dir = os.path.dirname(type_tag.text)
-
+    if not src_dir:
+        src_dir = os.path.dirname(vrt_file)
+    
```

Comment:
It would be helpful to write a comment here on the reason.

Reply:
When I create a multilooked vrt the fields are slightly off and that XML tag returns empty. I can add a comment.